tests/integration/issues/github_1468/flow.yml

```diff
@@ -0,0 +1,17 @@
+!Flow
+with:
+  prefetch: 10
+  read_only: true  # better add this in the query time
+pods:
+  filter_chunk_1:
+    uses: pods/filter_chunk_1.yml
+    read_only: true
+  filter_chunk_2:
+    uses: pods/filter_chunk_1.yml
+    read_only: true
+    needs: [gateway]
+  merge_route_process:
+    uses: _forward
+    parallel: 2
+    uses_before: _merge_chunks
```

Comment:
is it intentional, you could write `uses: _merge_chunks`

is it suggesting the problem only exists when you set both?

Reply:
`uses` could be anything else. 
I want `_merge_chunks` to accumulate all the `chunks` from the 2 incoming paths in the HeadPea, but at the same time HeadPea must handle Routing.

Reply:
but why a `needs` here does not fit your need?

```yaml
  merge_route_process:
    uses: _merge_chunks
    parallel: 2
#    uses_before: _merge_chunks
    needs: [filter_chunk_1, filter_chunk_2]

```

Reply:
ah will this merge everything? then yes I will test tomorrow. But then the load balancing could fail to send every pair of chunks to the same shard right?

Reply:
no i mean you can always separate it into two pods, e.g.
```yaml
!Flow
version: '1.0'
pods:
  - name: filter_chunk_1
    uses: pods/filter_chunk_1.yml
    read_only: true
  - name: filter_chunk_2
    uses: pods/filter_chunk_1.yml
    read_only: true
    needs: [gateway]
  - needs: [filter_chunk_1, filter_chunk_2]
  - name: merge_route_process
    parallel: 2

```

Reply:
ok yes good. However there is big way for improvement. The big issue that I see is that then just changing the amount of parallelization to the same Pod is not enough which adds complexity to the user